ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/AddItemsToPortal/AddItemsToPortal.h

```diff
@@ -47,6 +47,7 @@ class AddItemsToPortal : public QQuickItem
   Q_INVOKABLE void authenticatePortal();
   Q_INVOKABLE void addItem();
   Q_INVOKABLE void deleteItem();
+  void fetchItem();
```

Comment:
Is this only called internally from C++ code? If so, consider making it private.